atd-etl/data_model/README.md

```diff
@@ -35,23 +35,60 @@ Create your environment by saving a copy of the `env_template` file as `.env`. T
 ```
 ENV=dev
 HASURA_GRAPHQL_ENDPOINT="http://localhost:8084/v1/graphql"
-AWS_ACCESS_KEY_ID= 
+AWS_ACCESS_KEY_ID=
 AWS_SECRET_ACCESS_KEY=
 BUCKET_NAME=
 EXTRACT_PASSWORD=
 ```
 
-## Usage
+## Usage examples
 
 The only script that should be run directly is `cris_import.py`. It supports the following CLI args:
 
 ```shell
-  --csv              Only process CSV files
-  --pdf              Only process CR3 pdfs
+  --csv              Process CSV files. At least one of --csv or --pdf is required.
+  --pdf              Process CR3 pdfs. At least one of --csv or --pdf is required.
```

Comment:
im liking this update you made to require --csv or --pdf and you can use both arguments at once. its more intuitive :))